src/VisualStudio/TestUtilities/Remoting/WorkspaceWrapper.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.CodeAnalysis.Editor.Options;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.VisualStudio.LanguageServices;
+
+namespace Roslyn.VisualStudio.Test.Utilities.Remoting
+{
+    internal class WorkspaceWrapper : MarshalByRefObject
+    {
+        private readonly VisualStudioWorkspace _workspace;
+
+        public static WorkspaceWrapper Create()
+        {
+            var visualStudioWorkspace = RemotingHelper.VisualStudioWorkspace;
+            return new WorkspaceWrapper(visualStudioWorkspace);
+        }
+
+        private WorkspaceWrapper(VisualStudioWorkspace workspace)
+        {
+            _workspace = workspace;
+        }
+
+        public bool UseSuggestionMode
+        {
+            get
+            {
+                return OptionService.GetOption(EditorCompletionOptions.UseSuggestionMode);
+            }
+
+            set
+            {
+                if (value == UseSuggestionMode)
+                {
+                    return;
+                }
+
+                var options = OptionService.GetOptions();
+                options = options.WithChangedOption(EditorCompletionOptions.UseSuggestionMode, value);
+                OptionService.SetOptions(options);
+
+                // Recalculate the completion session, if present.
```

Comment:
This is reimplementing product behavior. if the setting isn't set how we want it, should we just send the toggle command to test the product behavior?